src/Features/CSharp/Portable/Completion/KeywordRecommenders/LoadKeywordRecommender.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.Extensions.ContextQuery;
+
+namespace Microsoft.CodeAnalysis.CSharp.Completion.KeywordRecommenders
+{
+    internal class LoadKeywordRecommender : AbstractSyntacticSingleKeywordRecommender
+    {
+        public LoadKeywordRecommender()
+            : base(SyntaxKind.LoadKeyword, isValidInPreprocessorContext: true)
+        {
+        }
+
+        protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken)
+        {
+            var syntaxTree = context.SyntaxTree;
+            return
+                syntaxTree.IsInteractiveOrScript() &&
+                syntaxTree.IsBeforeFirstToken(position, cancellationToken) &&
+                context.IsPreProcessorKeywordContext;
```

Comment:
In the end it might not matter, but it could be faster to move `IsBeforeFirstToken()` to be the last item in this list since it is a potentially complicated check that has to walk the syntax tree where `IsInteractiveOrScript()` and `IsPreProcessorKeywordContext` are simple checks and should short-circuit much faster.